src/main/scala/com/gu/dynamodbswitches/Switches.scala

```diff
@@ -35,6 +34,27 @@ trait Switches extends Logging {
         error(s"Encountered Amazon service error when trying to update switches from DynamoDB: ${exception.getMessage}")
     }
   }
+
+    def updateDynamo(switches : List[Switch]) = {
+    val listWR = switches.map(switch => {
+      new WriteRequest(new PutRequest(Map("name" -> switch.toStringAttribute(), "enabled" -> switch.toNumAttributeValue()).asJava))
+    }).asJava
+
+    try {
+      dynamoDbClient.batchWriteItem(Map(dynamoDbTableName -> listWR).asJava)
+    } catch {
+      case e: Exception => println(e)
+      case _ => println("Something went wrong")
```

Comment:
Ideally we'd log this, but I can see adding support for a logger might be easier in a separate PR.